docs/guides/darwin.md

```diff
@@ -306,8 +306,17 @@ Example:
 -   Please refer to the iOS/iPadOS 16.1
```

Comment:
is there actually value in spelling out each release note?

Can't we just link to https://developer.apple.com/documentation/ios-ipados-release-notes ?

Reply:
In perpetuity probably not, but perhaps a rolling window is useful for folks.